assets/terraform-modules/aws/flatcar-linux/kubernetes/cl/controller.yaml.tmpl

```diff
@@ -170,6 +170,7 @@ storage:
           ETCD_PEER_CERT_FILE=/etc/ssl/etcd/etcd/peer.crt
           ETCD_PEER_KEY_FILE=/etc/ssl/etcd/etcd/peer.key
           ETCD_PEER_CLIENT_CERT_AUTH=true
+          ${etcd_arch_options}
```

Comment:
I think generic etcd_env_variables looping over string-string map would be better.

Reply:
I've changed the name. I'm not sure what the string-string map would look like... Again, this is copied from the packet code and it probably shouldn't drift?

Reply:
Something like:
```
%{ for key, val in etcd_env_variables ~}
${key}=${val}
%{ endfor ~}
```

>Again, this is copied from the packet code and it probably shouldn't drift?

Hmm... Either way is fine then I guess.